application-admintools-ui/src/main/resources/AdminTools/Code/HealthCheckResources.xml

```diff
@@ -51,13 +53,16 @@
 
 #macro (getHealthCheckResources)
   #set ($jobMinJsURL = $xwiki.getSkinFile('uicomponents/job/job.min.js'))
-  #set ($healthCheckJobRequest = $services.admintools.createJobRequest($services.wiki.getCurrentWikiId()))
-  #set ($healthJobStatus = $services.job.getJobStatus($healthCheckJobRequest.id))
+  #set ($jobId = ['adminTools', 'healthcheck', $services.wiki.getCurrentWikiId()])
```

Comment:
I take it that you can only run a single health check job per wiki? In that case you should implement job grouping so that the second job on the same wiki waits for the first to finish. I also don't see why you can't set this ID from the script service. Is there a use case where the user needs to specify the job ID differently?